### PR TITLE
fix(blog): fix typos and remove extra spaces

### PR DESCRIPTION
**Pull request by VoltrexKeyva**
Fix typos and remove extra spaces in the Node.js v20 announcement blog.

### CODE REVIEW

**Comment by VoltrexKeyva** [comment]
The tests failures seem unrelated to my changes.

**Review by ovflowd — APPROVED** [review]
LGTM!

**Comment by ovflowd** [comment]
cc @nodejs/releasers

File: pages/en/blog/announcements/v20-release-announce.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ The stable test runner includes the building blocks for authoring and running te
  * `node --test` for running multiple test files in parallel
 
 The test runner also includes some parts that are not yet stable, including reporters and code coverage
```
Comment (tniessen) [inline pages/en/blog/announcements/v20-release-announce.md]:
```suggestion
The test runner also includes some parts that are not yet stable, including reporters and code coverage.
```

**Review by tniessen — APPROVED** [review]

**Review by RafaelGSS — APPROVED** [review]

**Review by shanpriyan — APPROVED** [review]
LGTM! :shipit:

**Review by aymen94 — APPROVED** [review]

**Pull request merged**